spyder_kernels/utils/tests/test_iofuncs.py

```diff
@@ -350,6 +351,10 @@ def test_save_load_hdf5_files(tmp_path):
     assert repr(iofuncs.load_hdf5(h5_file)) == repr(expected)
 
 
+@pytest.mark.skipif(
+    sys.version_info < (3, 10, 0),
+    reason="pydicom 3.0 is incompatible with Python < 3.10"
+)
```

Comment:
```suggestion
@pytest.mark.skipif(
    os.environ.get("USE_CONDA") == "true",
    reason="Pydicom is not installed correctly in Conda envs"
)
```

The test should run as expected with pip packages for Python 3.10+ and 3.9-, which is much better.